ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/DX11vertex.cpp

```diff
@@ -248,6 +249,8 @@ void vertex_color(int buffer, int color, double alpha) {
 }
 
 void vertex_submit(int buffer, int primitive, unsigned start, unsigned count) {
+  draw_batch_flush(batch_flush_deferred);
```

Comment:
Please reevaluate whether this function requires a flush (I am not certain myself).

Reply:
It is needed, consider the following scenario:
* The user sets the batch flush mode to deferred (won't flush until state change)
* The user calls `draw_sprite` a few times (which is batched together but waiting to be drawn)
* The user then calls `vertex_submit` without a texture (no state change occurs) and the vertex buffer is drawn immediately
* The `draw_sprite` calls still haven't been flushed... and if they are flushed beyond this point, they will appear "above" the vertex buffer the user just drew